source/includes/steps-define-a-function-import-export.yaml

```
@@ -1,153 +1,103 @@
-title: Export Your Realm Application
+title: Pull Your Realm Application
 ref: export-your-realm-application
 content: |
   To create new function with :doc:`realm-cli
```

```suggestion
  To create a new function with :doc:`realm-cli
```

I realize this isn't part of your PR, but this should prob be fixed.